.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
 install:
     - pip install -r requirements.txt
     - pip install coveralls
+    - pip install -U drf-yasg
```

Comment:
@VivianDoreen Please remove this from here.

Reply:
OK, let me do that

Reply:
Am done removing "- pip install -U drf-yasg"